plugins/offline-storage.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Offline Storage plugin
+ * 
+ * Stores errors failed to get send and try to send them when
+ * Networkf comes back or on init
```

Comment:
(sorry, this is a drive-by comment)

is this a typo, `Networkf`? should be `Network`, right?